### ISSUE TITLE
EDM PointLeaderboard component question

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#5425]
Just wondering about correct component behavior
For example i have 10 users with points, i setup to see 5 users in EDM leaderboard list
Email is sending to the user which are 9th in list
It receive a email where he can see that he i 6th.
![image](https://user-images.githubusercontent.com/1945230/39861918-d3fafd00-544a-11e8-9d24-41196d3b9679.png)
So the questions are:
1. Should user see himself in list when he doesn't fall into limit? (my opinion - he should see himself)
2. Should it give a correct position of his points amount, e.g. 1,2,3,4,5,9?
http://vladtest51e.admin.kademi-ci.co/groupEmails/16081613/#messages-tab

**Comment by bradmac** [comment]
> Should user see himself in list when he doesn't fall into limit? (my opinion - he should see himself)

Yes, generally they should, although i think we should have an option

> Should it give a correct position of his points amount, e.g. 1,2,3,4,5,9?

Yes, he should his actual leaderboard position, so that looks like a bug
